CONTRIBUTING.md

```diff
@@ -99,6 +99,30 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
 
 The full Meilisearch release process is described in [this guide](https://github.com/meilisearch/engine-team/blob/main/resources/meilisearch-release.md). Please follow it carefully before doing any release.
 
+### How to publish a prototype
+
+Depending on the developed feature, you might need to provide a prototyped version of Meilisearch to make it easier to test by the users.
+
+The prototype name must follow this convention: `prototype-X-Y` where
+- `X` is the feature name
+- `Y` is the version of the prototype, starting from `0`.
+
+Example: `prototype-auto-resize-0`.
+
+Steps to create a prototype:
+
+1. In your terminal, go to the last commit of your branch (the one you want to provide as a prototype).
+2. Create a tag following the convention: `git tag prototype-X-Y`
+3. Push the tag: `git push origin prototype-X-Y`
+4. Check the [Docker CI](https://github.com/meilisearch/meilisearch/actions/workflows/publish-docker-images.yml) is now running.
+
+🐳 Once the CI has finished to run (~1h30), a Docker image named `prototype-X-Y` will be available on [DockerHub](https://hub.docker.com/repository/docker/getmeili/meilisearch/general). People can use it with the following command: `docker run -p 7700:7700 -v $(pwd)/meili_data:/meili_data getmeili/meilisearch:prototype-X-Y`. <br>
```

Comment:
```suggestion
🐳 Once the CI has finished to run (~1h30), a Docker image named `prototype-X-Y` will be available on [DockerHub](https://hub.docker.com/repository/docker/getmeili/meilisearch/general). People can use it with the following command: `docker run -p 7700:7700 -v $(pwd)/meili_data:/meili_data getmeili/meilisearch:prototype-X-Y`.
```

Reply:
I don't want to remove teh `<br>` because it enforces to have a newline. Otherwise this paragraph and the next one are stuck together, what I don't want 😇

Reply:
fyi if you put two newlines it's supposed to be the equivalent of a `<br/>`.
Personally, I don't have a preference

Reply:
> fyi if you put two newlines it's supposed to be the equivalent of a <br/>.

Not completely. Two whitespaces at the end of a line are the same as a `<br/>`. And I hate it, and my code editor hates it, so I'm with curqui on the `<br/>` here.

Two newlines are the same as starting a new paragraph `<p>...`, which is generally sufficient in my book, but sometimes you really just want a single linebreak.